CHANGELOG.md

```diff
@@ -1,5 +1,128 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## v5.0.0-alpha.24
+
+<!-- generated comparing v5.0.0-alpha.23..next -->
+
+_Jan 25, 2021_
+
+Big thanks to the 21 contributors who made this release possible: @eps1lon, @mbrookes, @mnajdova, @greguintow, @praveenkumar-kalidass, @oliviertassinari, @lukaselmer, @povilass, @eladmotola, @kutnickclose, @artola, @vinyldarkscratch, @KarimMokhtar, @mxsph, @tonysepia, @d3mp, @CyanoFresh, @xs9627, @vicasas, @TimonPllkrn, @duganbrett
+
+Here are some highlights:
+
+- Converted TODO components to emotion
```

Comment:
Probably it's better from this release to start just state here number of components we converted, and people can take a look on the whole release notes to see which components. What do you think?

Reply:
That was the idea :grin: 

```diff
- Converted TODO components to emotion
+ Converted 10 components to emotion
```

Reply:
Maybe include a % of the progress, to give developers an idea of the global picture?

Reply:
That's what I did with the TS demo effort. Up to @mnajdova if she wants to do the same.

Reply:
That makes sense too 👍

Reply:
49/128 done, which is arond 38%